source/tyr/tyr/default_settings.py

```diff
@@ -128,6 +132,11 @@
         'schedule': schedules.crontab(hour=2, minute=30),  # Task is executed daily at 2H30
         'options': {'expires': 120}
     },
+    'purge-old-job-every-week': {
+        'task': 'tyr.tasks.purge_jobs',
+        'schedule': schedules.crontab(1,0, day_of_month='1'), # Task is executed at at 1H00 on the first day of every month
```

Comment:
We mentionned once a week, not once a month (and we better choose a random day, or a low-charge day)

Reply:
the name of the task is correct but not the frequency... Will change it.